### DISCUSSION TITLE
Will this become a package to utilize in other projects?

### DISCUSSION THREAD

**Discussion by NewtFDevelopment** [miragedmuk/ASV#30] [Q&A]
Hey,

Looks like some awesome coding went into this! I was curious if this would be something that can be used as a package?

**Comment by miragedmuk** [comment]
Not sure what you mean for it to be used "as a package"?

**Comment by miragedmuk** [comment]
Ohh - I see what you mean now.  

It's all very hacky and I wouldn't be comfortable providing it as a Nuget package for others to use in their projects but you're free to use the code and binaries however you wish.
